src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,44 @@
+package edu.kis.vh.nursery;
+
+/**
+ *
+ */
+public class DefaultCountingOutRhymer {
+
+    private static final int MAX_SIZE = 12;
+    private static final int EMPTY = -1;
+    private static final int FULL = 11;
+    private final int[] numbers = new int[MAX_SIZE];
+
+    public int getTotal() {
+        return total;
+    }
+
+    private int total = EMPTY;
+
+    protected void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+    protected boolean callCheck() {
+        return total == EMPTY;
```

Comment:
This constant should be different (separate) than the one in peekaboo and countOut methods.

Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).